packages/app/src/server/models/config.ts

```diff
@@ -245,6 +245,7 @@ schema.statics.getLocalconfig = function(crowi) {
     globalLang: crowi.configManager.getConfig('crowi', 'app:globalLang'),
     pageLimitationL: crowi.configManager.getConfig('crowi', 'customize:showPageLimitationL'),
     pageLimitationXL: crowi.configManager.getConfig('crowi', 'customize:showPageLimitationXL'),
+    currentBrandLogo: crowi.configManager.getConfig('crowi', 'customize:currentBrandLogo'),
```

Comment:
Isn't this supposed to be `customizedLogoSrc` ?

https://github.com/weseek/growi/pull/5709#discussion_r910604931

Reply:
[As replied here](https://github.com/weseek/growi/pull/5709#discussion_r913453041)